pkg/login/social/social.go

```diff
@@ -96,12 +102,20 @@ func NewOAuthService() {
 			TlsClientKey:       sec.Key("tls_client_key").String(),
 			TlsClientCa:        sec.Key("tls_client_ca").String(),
 			TlsSkipVerify:      sec.Key("tls_skip_verify_insecure").MustBool(),
+			GroupMappingsFile:  sec.Key("group_mappings_file").String(),
 		}
 
 		if !info.Enabled {
 			continue
 		}
 
+		groupMappingsConfig, err := readConfig(info.GroupMappingsFile)
+		if err != nil {
+			log.Error(3, "Error reading group mappings config: %s", err.Error())
```

Comment:
```suggestion
			oauthLogger.Error("Failed to read group mappings file", "provider", name, "error", err)
```

Reply:
This change was committed, only different is I kept `file` as one of the items logged just so logs show which file failed to be read.